mmc-github/src/main/resources/application.yml

```diff
@@ -0,0 +1,4 @@
+github.project:
```

Comment:
"application" is what comes from Spring by default just like application.properties. I would stick to the default for not addtional confguration.